packages/manager/modules/sms/src/sms/options/smppParameter/allowed-ips/allowed-ips.html

```diff
@@ -0,0 +1,88 @@
+<div class="col-sm-4">
```

Comment:
Same as https://github.com/ovh/manager/pull/8512/files#r1040709971

Reply:
It's fixed. I move this class.